x-pack/plugins/security_solution/public/risk_score/containers/feature_status/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback, useEffect, useMemo } from 'react';
+import { useMlCapabilities } from '../../../common/components/ml/hooks/use_ml_capabilities';
+import { REQUEST_NAMES, useFetch } from '../../../common/hooks/use_fetch';
+import { RiskQueries } from '../../../../common/search_strategy';
+import { getRiskScoreIndexStatus, RiskEntity } from './api';
+
+interface RiskScoresFeatureStatus {
+  error: unknown;
+  // Is transform index an old version?
+  isDeprecated: boolean;
+  // does the transform index exist?
+  isEnabled: boolean;
+  // is the user's license platinum?
+  isLicenseValid: boolean;
```

Comment:
@angorayc heads up that this function name changed from `useRiskScoreDeprecated` to `useRiskScoreFeatureStatus` since it now returns `isDeprecated`, `isLicenseValid`, and `isEnabled`